src/helpers/utils.js

```diff
@@ -34,24 +35,19 @@ export const isEmpty = obj => {
     return true;
   }
 
-  // Assume if it has a length property with a non-zero value
-  // that property is correct.
-  if (obj.length > 0) {
-    return false;
+  if (obj.hasOwnProperty('length')) {
+    return obj.length === 0;
   }
 
-  if (obj.length === 0) {
-    return true;
+  if (typeof obj == 'object') {
+    return Object.getOwnPropertyNames(obj).length === 0;
   }
 
-  // If it isn't an object at this point
-  // it is empty, but it can't be anything *but* empty
-  // Is it empty?  Depends on your application.
-  if (typeof obj !== 'object') {
-    return true;
+  if (typeof obj == 'number') {
+    return isNaN(obj);
```

Comment:
You have not testing this.

Why   !isNaN    should be not empty?

Reply:
a blin, you right

Reply:
when its NaN its empty (isNaN(obj) === true) thats its return